src/test/java/LoginTests.java

```diff
@@ -1,55 +1,62 @@
-
-        provideEmail(email);
-        providePassword(password);
-        clickSubmit();
-
-        Thread.sleep(2000);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
-
-    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class)
-    public void loginEmptyEmailPassword(String email, String password) throws InterruptedException {
-        provideEmail(email);
-        providePassword(password);
-        clickSubmit();
-
-        Thread.sleep(2000);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
-    
+//    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class, enabled = true, priority = 0, description = "Login with invalid email and valid password")
+//    public void loginInvalidEmailValidPassword(String email, String password) throws InterruptedException {
+//
+//        provideEmail(email);
+//        providePassword(password);
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
+//
+//    @Test (dataProvider = "IncorrectLoginData", dataProviderClass = BaseTest.class)
+//    public void loginEmptyEmailPassword(String email, String password) throws InterruptedException {
+//        provideEmail(email);
+//        providePassword(password);
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url);
+//    }
+//
+//    @Test (enabled = true, priority = 1, description = "Login with valid email and valid password")
+//    public void loginValidEmailPassword(){
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("te$t$tudent");
+//        clickSubmit();
+//        isAvatarDisplayed();
+//    }
+//
+//    @Test (enabled = true, priority = 3, description = "Login with valid email and empty password")
+//    public void loginValidEmailEmptyPassword() throws InterruptedException {
+//
+//        navigateToPage();
+//        provideEmail("demo@class.com");
+//        providePassword("");
+//        clickSubmit();
+//
+//        Thread.sleep(2000);
+//        Assert.assertEquals(driver.getCurrentUrl(), url); //https://qa.koel.app/
+//    }
+//    public void isAvatarDisplayed() {
+//        WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+//        Assert.assertTrue(avatarIcon.isDisplayed());
+//    }
     @Test (enabled = true, priority = 1, description = "Login with valid email and valid password")
-    public void loginValidEmailPassword(){
-
-        navigateToPage();
-        provideEmail("demo@class.com");
-        providePassword("te$t$tudent");
-        clickSubmit();
-        isAvatarDisplayed();
+    public void loginValidEmailValidPass() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.loginWithValidCredentials();
```

Comment:
Use in test methods, the following logical split:   GIVEN – WHEN - THEN

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/dfacfd06-4b50-4d5a-a52d-86412170fc51)